athena-msk/athena-msk.yaml

```diff
@@ -0,0 +1,149 @@
+  CertificatesS3Reference:
+    Description: 'The S3 bucket reference where keystore and truststore certificates are uploaded,applicable for TLS auth '
+    Type: String
+  GlueRegistryArn:
+    Description: 'AWS GLUE registry ARN to be provided where topic schemas has been uploaded'
+    Type: String
+  LambdaRoleARN:
+    Description: "(Must for auth type IAM) A custom role to be used by the Connector lambda"
+    Default: ""
+    Type: String
+
+Conditions:
+  NotHasLambdaRole: !Equals [!Ref LambdaRoleARN, ""]
+
+Resources:
+  JdbcConnectorConfig:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      Environment:
+        Variables:
+          disable_spill_encryption: !Ref DisableSpillEncryption
+          spill_bucket: !Ref SpillBucket
+          spill_prefix: !Ref SpillPrefix
+          secrets_manager_secret: !Ref SecretNamePrefix
+          certificates_s3_reference: !Ref CertificatesS3Reference
+          kafka_endpoint: !Ref KafkaEndpoint
+          auth_type: !Ref AuthType
+          glue_registry_arn:  !Ref GlueRegistryArn
+      FunctionName: !Ref LambdaFunctionName
+      Handler: "com.amazonaws.athena.connectors.msk.AmazonMskCompositeHandler"
+      CodeUri: "./target/athena-msk-2022.47.1.jar"
+      Description: "Enables Amazon Athena to communicate with MSK and Kafka clusters"
+      Runtime: java11
+      Timeout: !Ref LambdaTimeout
+      MemorySize: !Ref LambdaMemory
+      Role: !If [NotHasLambdaRole, !GetAtt FunctionRole.Arn, !Ref LambdaRoleARN]
+      VpcConfig:
+        SecurityGroupIds: !Ref SecurityGroupIds
+        SubnetIds: !Ref SubnetIds
+  FunctionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      ManagedPolicyArns:
+        - "arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole"
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - "sts:AssumeRole"
+
+  FunctionExecutionPolicy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyName: FunctionExecutionPolicy
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Action:
+              - secretsmanager:GetSecretValue
+            Effect: Allow
+            Resource: !Sub 'arn:${AWS::Partition}:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:${SecretNamePrefix}*'
+          - Action:
+              - logs:CreateLogGroup
+            Effect: Allow
+            Resource: !Sub 'arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:*'
+          - Action:
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+            Effect: Allow
+            Resource: !Sub 'arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/${LambdaFunctionName}:*'
+          - Action:
+              - athena:GetQueryExecution
+              - s3:ListAllMyBuckets
```

Comment:
I noticed that the following actions are now missing:
```
- kms:Decrypt
- glue:*
```


Is this intentional?